packages/caravan-bitcoin/src/__tests__/script.test.ts

```diff
@@ -1,5 +1,6 @@
-import { scriptToOps, scriptToHex } from "./script";
-import { TEST_FIXTURES } from "./fixtures";
+import { describe, it, expect } from "vitest";
```

Comment:
these shouldn't be necessary with the globals.

Reply:
Can you please elaborate on this ... don't we need the fixtures and script functions to make these tests run ?

Reply:
changes [done](https://github.com/caravan-bitcoin/caravan/pull/135/commits/f2bb57fe1224fa6dcad269f6b9843f023a5ef899#diff-25c8a3137d93ae90bef805837c3d0bd5075d36b4b00ff965deb495aa8baddc13R1)